lib/script.js

```diff
@@ -83,7 +83,7 @@ Script.prototype.run = function (ctx, domain, fn) {
       callbackCount++;
     });
 
-    events.on('finishCallback', function() {
+    events.on('finishCallback', function () {
       callbackCount--;
```

Comment:
if the wrap function calls without Promise or callback,
the ```callbackCount``` may be wrong, 
as the number of call to ```addCallback``` and ```finishCallback``` is not in pair?